provisioning/windows-provision.ps1

```diff
@@ -174,7 +174,7 @@ $downloads = [ordered]@{
         'cleanupLocal' = 'true';
     };
     'jq' = @{
-        'url' = 'https://github.com/stedolan/jq/releases/download/jq-{0}/jq-win64.exe'  -f $env:JQ_VERSION;
+        'url' = 'https://github.com/jqlang/jq/releases/download/jq-{0}/jq-win64.exe'  -f $env:JQ_VERSION;

```

Comment:
I would add something about this change in the PR body.

Was it required for this PR or could it have been its own?

Reply:
Thanks for the explanation. I still would add a mention of this change in the PR body 😉